tests/galsim_test_helpers.py

```diff
@@ -361,7 +361,8 @@ def all_obj_diff(objs):
         for j, objj in enumerate(objs):
             if i == j:
                 continue
-            assert obji != objj, "Found equivalent objects {} == {}".format(obji, objj)
+            assert obji != objj, ("Found equivalent objects {} == {} at indices {} and {}"
+                                  .format(obji, objj, i, j))
```

Comment:
For compatibility with python 2.6, these `{}` need to have numbers in them.  OTOH, maybe it's finally time to drop our support for python 2.6?  I still have one old system that has 2.6, which I usually only test on before a release (and fix little things like this).  But perhaps we shouldn't bother anymore.  Probably everyone who uses GalSim has 2.7 on the machines where they want to be using GalSim.